tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -640,8 +641,7 @@ mapOffsetToNewerSnapshot(unsigned Offset,
 static void mapLocToLatestSnapshot(
     SwiftLangSupport &Lang, LocationInfo &Location,
     ArrayRef<ImmutableTextSnapshotRef> PreviousASTSnaps) {
-  auto EditorDoc = Lang.getEditorDocuments()->findByPath(Location.Filename,
-                                                         /*IsRealpath=*/true);
+  auto EditorDoc = Lang.getEditorDocuments()->findByPath(Location.Filename);
```

Comment:
I think you want to leave the `IsRealpath` in here, possibly renamed. `findByPath` was changed to also use `resolvePathSymlinks`

Reply:
The reason I didn't keep it is that in some call sites we don't have a FileSystem available, in which case we could only support `IsRealPath == false` by using the real filesystem, and it's not clear that's what we want. I also thought of naming the argument `FileSystemForRealPath` if that's better.

Reply:
Ah, I missed you made that change. Makes sense 👍 `FileSystem` is fine with me. Just have to be careful that all the places that previously didn't take an argument (and thus were resolved within `findByPath`) now do.

Reply:
Yes, I audited all call sites!